examples/select-table-rows-by-content/index.js

```diff
@@ -0,0 +1,20 @@
+import { Selector } from 'testcafe'
+
+fixture `Check table row content`
+    .page('./index.html')
+
+test('Check table row content', async t => {
+    const rowData = {
+        id: '2',
+        firstName: 'Oliver',
+        lastName: 'Black'
+    }
+    const row = Selector('td').withText(rowData.id).parent();
+
+    await t.click(row);
```

Comment:
```js
await t
     .click(row)
     .expect(row.child('td').nth(0).innerText).eql(rowData.id)
     .expect(row.child('td').nth(1).innerText).eql(rowData.firstName)
     .expect(row.child('td').nth(2).innerText).eql(rowData.lastName)
```

Reply:
Also, add check that clicked row is highlighted (`row.hasClass('highlighted')`);

Reply:
edited